ghost/posts-service/lib/PostsService.js

```
@@ -21,39 +21,119 @@ const messages = {
+        if (shouldNotIncludeAttr(options, 'custom_excerpt')) {
+            delete postAttrs.custom_excerpt;
+        }
+        if (shouldIncludeAttr(options, 'plaintext')) {
+            postAttrs.plaintext = model.get('plaintext');
+        }
+        if (shouldIncludeAttr(options, 'excerpt')) {
+            postAttrs.excerpt = model.get('custom_excerpt') || model.get('plaintext')?.substring(0, 500) || null;
+        }
+        if (shouldIncludeAttr(options, 'reading_time')) {
+
+        }
```

ESLint

[error] Empty block statement.

(no-empty)